boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -119,6 +119,7 @@
 				line-name = "exp_sel";
```

Comment:
@JiafeiPan  care to explain why this happens?

Reply:
exp_sel signal is used by flexcan,  refer to https://github.com/zephyrproject-rtos/zephyr/pull/70201, @CkovMk can correct me if it is not the case.